share/org.getmonero.Monero.yaml

```diff
@@ -40,46 +36,46 @@ modules:
       - ./b2 -j$FLATPAK_BUILDER_N_JOBS install variant=release --layout=system
     sources:
       - type: archive
-        url: https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.gz
-        sha256: 66a469b6e608a51f8347236f4912e27dc5c60c60d7d53ae9bfe4683316c6f04c
+        url: https://archives.boost.io/release/1.84.0/source/boost_1_84_0.tar.gz
+        sha256: a5800f405508f5df8114558ca9855d2640a2de8f0445f051fa1c7c3383045724
 
   - name: libunbound
     config-opts:
       - --with-libunbound-only
     sources:
       - type: archive
-        url: https://nlnetlabs.nl/downloads/unbound/unbound-1.17.1.tar.gz
-        sha256: ee4085cecce12584e600f3d814a28fa822dfaacec1f94c84bfd67f8a5571a5f4
+        url: https://nlnetlabs.nl/downloads/unbound/unbound-1.22.0.tar.gz
+        sha256: c5dd1bdef5d5685b2cedb749158dd152c52d44f65529a34ac15cd88d4b1b3d43
 
   - name: libsodium
     sources:
       - type: archive
-        url: https://github.com/jedisct1/libsodium/releases/download/1.0.18-RELEASE/libsodium-1.0.18.tar.gz
-        sha256: 6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1
+        url: https://github.com/jedisct1/libsodium/releases/download/1.0.20-RELEASE/libsodium-1.0.20.tar.gz
+        sha256: ebb65ef6ca439333c2bb41a0c1990587288da07f6c7fd07cb3a18cc18d30ce19
 
   - name: libusb
     sources:
       - type: archive
-        url: https://github.com/libusb/libusb/releases/download/v1.0.26/libusb-1.0.26.tar.bz2
-        sha256: 12ce7a61fc9854d1d2a1ffe095f7b5fac19ddba095c259e6067a46500381b5a5
+        url: https://github.com/libusb/libusb/releases/download/v1.0.27/libusb-1.0.27.tar.bz2
+        sha256: ffaa41d741a8a3bee244ac8e54a72ea05bf2879663c098c82fc5757853441575
 
   - name: hidapi
     buildsystem: cmake-ninja
     config-opts:
       - -DCMAKE_BUILD_TYPE=Release
     sources:
       - type: archive
-        url: https://github.com/libusb/hidapi/archive/hidapi-0.13.1.tar.gz
-        sha256: 476a2c9a4dc7d1fc97dd223b84338dbea3809a84caea2dcd887d9778725490e3
+        url: https://github.com/libusb/hidapi/archive/hidapi-0.14.0.tar.gz
```

Comment:
https://github.com/monero-project/monero-gui/blob/c957058860194b7c6113811556d8e3405a08a03a/Dockerfile.linux#L236

could bump to 15 #4534